colorbleed/plugins/maya/publish/collect_look.py

```diff
@@ -205,13 +187,7 @@ def collect_member_data(self, member, objset_members, instance_members,
             self.log.error("Node '{}' has no attribute 'cbId'".format(node))
             return
 
-        member_data = {"name": node, "uuid": lib.get_id(node)}
-
-        # Include components information when components are assigned
-        if components:
```

Comment:
I don't see a need to remove the `components` from collection; they are part of the sets data. Even if we'd recommend avoiding things like face assignments I don't see a reason to break this particular logic at this point. Unless there's a strong argument to support this removal let's keep it in for now and preserve this logic.